### PR TITLE
fix #39960 m.silisili.me

### LINKED ISSUE #39960: m.silisili.me
### Issue URL (Ads)
[http://m.silisili.me/anime/2170.html](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=http%3A%2F%2Fm.silisili.me%2Fanime%2F2170.html)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | value
--- | ---
Platform: | iOS
AdGuard version: | 2.1.2
Browser: | Safari
System wide filtering: | disabled
Simplified filters: | disabled
AdGuard DNS: | None
Filters: | EasyList China,<br>AdGuard Safari,<br>AdGuard Annoyances,<br>AdGuard Base,<br>CJX's Annoyance List,<br>AdGuard Japanese

### PR DESCRIPTION
**Pull request by zloyden**
https://github.com/AdguardTeam/AdguardFilters/issues/39960

### CODE REVIEW

File: MobileFilter/sections/specific_web.txt [hunk 1]
```diff
@@ -42,6 +42,7 @@ anwap.club,anwap.film,anwap.mobi##body > div[class] > a > strong
 ||mobile.donanimhaber.com/ads^
 !
 ficbook.net##.jsHideStickyRkl
+||etuch.cn^$image,domain=m.silisili.me
```
Comment (BlazDT) [inline MobileFilter/sections/specific_web.txt]: Not requested for me; also getting redirected several times until I hit silisili.cc

Comment (zloyden) [reply MobileFilter/sections/specific_web.txt]: When I open dev console it always redirects me to another page, but I only reproduced it (banner) once.

Comment (Alex-302) [reply MobileFilter/sections/specific_web.txt]: I don't see ads when Easylist China is enabled. But redirected to another site.

**Comment by accountforadg** [comment]
@Alex-302  @zloyden   
image

**Comment by zloyden** [comment]
That ad I can't reproduce anymore.

**Comment by zloyden** [comment]
According [user response](https://github.com/AdguardTeam/AdguardFilters/issues/39960#issuecomment-531238443) the rule in  fix works fine, but I can't reproduce it on my side.

**Review by Alex-302 — APPROVED** [review]

**Comment by Alex-302** [comment]
@accountforadg this button for our team only)

**Pull request merged**